actioncable/lib/action_cable/connection/identification.rb

```diff
@@ -2,7 +2,6 @@
 
 # :markup: markdown
 
-require "set"
```

Comment:
Removing `require 'set'` will result in two blank lines. It seems preferable to delete either the preceding or following blank line, leaving just one.

Reply:
Agreed. Let me remove them.

Reply:
Removed these two empty lines.